templates/main/signup.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Signup</title>
+        <link rel="stylesheet" href="css/main.css">
+        <script src="js/login.js" defer></script>
+    </head>
+    <body>
+    	<div id='container'>
+        <h1>Signup page</h1>
+    	 <form action="" method="POST">	  
+			  	
+			    <label for="uname"><b>Username</b></label>
+			    <input type="text" placeholder="Enter Username" name="uname" required>
+			    <br>
+			    <label for="email"><b>Email</b></label>
+			    <input type="text" placeholder="Enter Email" name="email" required>
+			    <br>
+			    <label for="psw"><b>Password</b></label>
+			    <input type="password" placeholder="Enter Password" name="psw" required>
+			    <br>
+			    <button type="submit" name='submitbutton'>Signup</button>
+			</form> 
+			<?php if (empty($params["msg"])): ?>
+		
+       <?php else: ?>
```

Comment:
An empty `if` with an `else` statement means that we can negate the condition in the `if` and remove the `else`, right?